.github/workflows/check-broken-links.yml

```
@@ -0,0 +1,30 @@
+name: Check for broken links
+
+on:
+  schedule:
+    - cron:  '0 10 * * *'
+
+jobs:
+  check_broken_links:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v4
+
+    - name: Check for broken links
+      run: |
+        touch valid_urls.txt redirected_urls.txt invalid_urls.txt
+        find . -type f \( -name "*.txt" -o -name "*.md" -o -name "*.py" -o -name "*.rst" \) \
+          \( \! -name 'CHANGES.md' \) \( \! -name 'requirements.txt' \) \( \! -name 'requirements-freeze.txt' \) |
+          xargs grep -Eo "\b(https?)://[-A-Za-z0-9+&@#/%?=~_|$!:,.;]*[A-Za-z0-9+&@#/%=~_|$]" |
+          sed 's/:/;/' |
+          xargs -n 1 -P 8 ./.github/workflows/check-url.sh
```

Since we're now only running the full link checker on daily jobs, perhaps we want to get rid of `-P 8` to have more coherent logging in this step? (The previous discussion of "3m vs. 13m" is kind of moot now.)

```suggestion
          xargs -n 1 ./.github/workflows/check-url.sh
```